core/jazz_scm-webhook/index.js

```diff
@@ -241,7 +241,7 @@ var gitlabScmContextDetails = function (eventKey, body, config) {
 			var branch = origins[2];
 			result.branch = branch;
 
-			if (body.before && parseInt(body.before, 10) === 0) {
+			if (body.before && body.before === '0000000000000000000000000000000000000000') {
```

Comment:
Can we use regex for this instead checking the value?

Reply:
`body.before && body.before.indexOf('0000000000') >= 0` This atleast checks there are atleast 10 zeroes in the string!

Reply:
@raghits Highly unlikely, but what if the sha starts with 10 0s? :)

Reply:
Closing this..